pkg/csi/service/common/util.go

```diff
@@ -303,6 +303,20 @@ func GetConfig(ctx context.Context) (*cnsconfig.Config, error) {
 	return cfg, err
 }
 
+// InitConfigInfo initializes the ConfigurationInfo struct
+func InitConfigInfo(ctx context.Context) (*cnsconfig.ConfigurationInfo, error) {
```

Comment:
What if this is called multiple times in the same thread?

Reply:
it is just returning a new instance of  *cnsconfig.ConfigurationInfo`, every time it is called. So no issue when called in threads.